### PR TITLE
Since openvpn and wireguard are both supported

### PR DESCRIPTION
**Pull request by ahmedsajid**
Updated the page to be VPN instead of OpenVPN

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
 -->
 <html>
 	<head>
-		<title>PiVPN: Simplest setup of OpenVPN</title>
```
Comment (coolapso) [inline index.html:9]: Maybe change to "PIVPN: Simplest way to setup a vpn"  .. What you think?

File: index.html [hunk 2]
```diff
@@ -58,7 +58,7 @@ <h1 id="logo"><a href="#">The PiVPN Project</a></h1>
 						<header class="major">
 							<h2>PiVPN</h2>
 							<p>
-								Simplest OpenVPN setup and configuration,<br />
```
Comment (coolapso) [inline index.html:61]: and here maybe ... 
"The simplest way to setup and configure a VPN"

**Review by coolapso — CHANGES_REQUESTED** [review]
well missed those ones when i updated the websites .... well .. thanks a lot for catching it up .. 
I have some suggestions for those changes .. check out the comments and let me know what you think .. if so feel free to change your PR and i will accept it.

File: index.html (hunk 2 above)
Comment (orazioedoardo) [reply index.html:61]: "setup and manage"

File: index.html (hunk 1 above)
Comment (ahmedsajid) [reply index.html:9]: sounds good.

File: index.html (hunk 2 above)
Comment (ahmedsajid) [reply index.html:61]: done

**Review by coolapso — COMMENTED** [review]
Looks good .. will merge and deploy

**Comment by coolapso** [comment]
Deployed.

**Pull request merged**